lib/snmplib/snmp_api.c

```diff
@@ -102,12 +102,8 @@
  * Returns the length of the encoded packet in out_length.  If an error
  * occurs, -1 is returned.  If all goes well, 0 is returned.
  */
-int
-snmp_build(session, pdu, packet, out_length)
-struct snmp_session *session;
-struct snmp_pdu *pdu;
-u_char *packet;
-int *out_length;
+int snmp_build(struct snmp_session *session,
+    struct snmp_pdu *pdu, u_char *packet, int *out_length)
```

Comment:
Restoring formatting:

```suggestion
int
snmp_build(struct snmp_session *session, struct snmp_pdu *pdu, u_char *packet, int *out_length)
```